.github/workflows/build.yml

```
@@ -0,0 +1,112 @@
+name: Build Godot with ECMAScript
+# TODO: Investigate making this on tag instead of push since builds take so long
+on: [push]
+
+# TODO: Need to also publish release builds without tools (for exporters) for linux, windows, mac, ios, android
+jobs:
+  build_linux:
+    name: Build Linux
+    runs-on: ubuntu-latest
+    container: 
+      # See how this image is built at https://github.com/Flux159/godot-docker
+      image: flux159/godot:1.0.0
+    steps:
+      - name: Get repositories
+        # TODO: Migrate this to use checkout action for ECMAScript.git rather than clone
+        run: | 
+          rm -rf /godot && cd /
+          git clone https://github.com/godotengine/godot.git 
+          cd ./godot
+          git checkout 3.2.2-stable
+          git clone https://github.com/GodotExplorer/ECMAScript.git ./modules/ECMAScript
+      - name: Build godot with ECMAScript module for Linux
+        run: |
+          cd /godot
+          scons -j8 p=x11
+          ls ./bin
+      - name: Publish artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: godotjs.x11.tools.64
+          path: /godot/bin/godot.x11.tools.64
+  build_windows:
+    name: Build Windows
+    runs-on: ubuntu-latest
+    container:
+      image: flux159/godot:1.0.0
+    steps:
+      - name: Get repositories
+        # TODO: Migrate this to use checkout action for ECMAScript.git rather than clone
+        # TODO: Windows builds broken with quickjs_debugger, so using checkout 01bc02
+        run: |
+          rm -rf /godot && cd /
+          git clone https://github.com/godotengine/godot.git 
+          cd ./godot
+          git checkout 3.2.2-stable
+          git clone https://github.com/GodotExplorer/ECMAScript.git ./modules/ECMAScript
+          cd modules/ECMAScript/ && git checkout 01bc02 && cd ../..
```

Don't checkout just let the build fail.